site/src/components/css-display/Accordian_view.tsx

```diff
@@ -0,0 +1,193 @@
+import React, { useEffect, useState } from "react";
+import {
+  Accordion,
+  AccordionHeader,
+  AccordionPanel,
+  Banner,
+  Link,
+  Tooltip,
+} from "@salt-ds/core";
+import { ColorBlock } from "docs/components/ColorBlock";
+import { FontSizeBlock } from "docs/components/FontSizeBlock";
+import { FontWeightBlock } from "docs/components/FontWeightBlock";
+import { CursorBlock } from "docs/components/CursorBlock";
+import { LetterSpacingBlock } from "docs/components/LetterSpacingBlock";
+import { LineBlock } from "docs/components/LineBlock";
+import { TextBlock } from "docs/components/TextBlock";
+import { OutlineBlock } from "docs/components/OutlineBlock";
+import { ShadowBlockCell } from "docs/components/ShadowBlock";
+import { Table } from "../mdx/table";
+import styles from "./Characteristics.module.css";
+import chars from "./descriptions";
+import { Heading4 } from "../mdx/h4";
+
+type CssVariableData = {
+  [key: string]: string;
+};
+
+const color: string[] = new Array(
+  "background",
+  "foreground",
+  "color",
+  "palette",
+  "borderColor",
+  "outlineColor",
+  "indicator"
+);
+const fontSize: string[] = new Array("fontSize", "minHeight");
+const fontWeight: string[] = new Array("fontWeight");
+const cursor: string[] = new Array("cursor");
+const letters: string[] = new Array("letterSpacing");
+const border: string[] = new Array(
+  "borderStyle",
+  "borderWidth",
+  "outlineWidth",
+  "outlineStyle"
+);
+const text: string[] = new Array("fontStyle");
+const outline: string[] = new Array("outline");
+const shadow: string[] = new Array("shadow");
+const align: string[] = new Array("textAlign");
+const textDecoration: string[] = new Array("textDecoration");
+const transform: string[] = new Array("textTransform");
+const fontFamily: string[] = new Array("fontFamily");
+const lineHeight: string[] = new Array("lineHeight");
+
+const MyComponent: React.FC<{ name: string }> = ({ name }) => {
```

Comment:
what is 'MyComponent'?

Reply:
changed the naming to BlockView,, basically decides which type of block to use when displaying the css vars